docs/installing_deis/azure.rst

```diff
@@ -0,0 +1,121 @@
+:title: Installing Deis on Microsoft Azure
+:description: How to provision a multi-node Deis cluster on Microsoft Azure
+
+.. _deis_on_azure:
+
+Microsoft Azure
+===============
+
+This section will show you how to create a 3-node Deis cluster on Microsoft Azure.
+
+Before you start, :ref:`get the Deis source <get_the_source>` and change directory into `contrib/azure`_
+while following this documentation.
+
+
+Install Python and Azure SDK for Python
+---------------------------------------
+
+The cluster creation tool uses Python and the Python Azure library to create a CoreOS cluster.
+If you haven't already, install these on your development machine:
+
+.. code-block:: console
+
+    $ brew install python
+    $ sudo pip install azure
+
+And check to make sure they are configured correctly:
```

Comment:
Latest is probably always best for the Azure CLI - I don't want to specify a potentially outdated version and have to update it.